hack/build.sh

```diff
@@ -50,7 +50,11 @@ function squash {
 dirs=${VERSION:-$VERSIONS}
 
 for dir in ${dirs}; do
-  IMAGE_NAME="${BASE_IMAGE_NAME}-${dir//./}-${OS}"
+  if [ "$dir" == "2.6" ]; then
```

Comment:
It it a part of releasing containers as a part of RHSCL. I don't know if even mongodb24 should follow "RHSCL" naming policy...
But I don't know much about this naming. @hhorak could explain more...